Work5

roma-outspin opened the pull request.

roma-outspin committed:
автоинкремент поля id

roma-outspin committed:
constructors

добавлены конструкторы в соответствии с заданием

roma-outspin committed:
авто свойства

roma-outspin committed:
снятие и внесение на счет

netaksa commented on GB-OOP-HW2/BankAccount.cs:
```diff
@@ -1,43 +1,71 @@
-        {
-            return _id;
-        }
+        private static int idGenerator = 1;
```
почему статическое?

roma-outspin replied:
Для того чтобы у каждого экземпляра был уникальный идентификатор

roma-outspin committed:
HW3

- перевод денег
- реверс строки
- работа со строками (емейл) (нужен рефакторинг)
